packages/angular_devkit/build_optimizer/src/transforms/scrub-file_spec.ts

```diff
@@ -173,6 +173,20 @@ describe('scrub-file', () => {
       expect(oneLine`${transform(input)}`).toEqual(oneLine`${output}`);
     });
 
+
+    fit('removes top-level Angular constructor parameters in es2015', () => {
```

Comment:
`fit`?

Reply:
Good catch 👍

Reply:
Fixed in https://github.com/angular/devkit/pull/170